src/tracing/server.coffee

```diff
@@ -0,0 +1,25 @@
+###
+This server response to ui on a /vizc end point and returns queried JSON as reponse
+###
+express = require 'express'
+bodyparser = require 'body-parser'
+varz = require 'express-varz'
+{conf} = require 'rainier/conf'
+port = conf.get 'vizc_server_port'
+vizJsonCreator = require './vizceral_json_file_creator_nodes'
+
+server = express()
+server.use bodyparser.json limit: '10mb'
+server.use bodyparser.urlencoded extended: true
+
+server.get '/vizc', (req, res, next) ->
```

Comment:
I know you're currently only exposing one function but please follow this setup:
https://github.com/seomoz/bainbridge/tree/master/lib/routes